app/services/foreman_webhooks/webhook_service.rb

```diff
@@ -65,7 +65,7 @@ def self.request(url:, payload: '', http_method: :GET, user: nil, password: nil,
       uri = URI.parse(url)
 
       request = Object.const_get("Net::HTTP::#{http_method.to_s.capitalize}").new(uri.request_uri)
-      request.basic_auth(user, password) if user && password
+      request.basic_auth(user, password) if !user.blank? && !password.blank?
```

Comment:
This is because new UI throws errors to console if the form has such fields `undefined` or `null`. So they should be `""`. But after form submit a webhook is being created with user/password being set alothugh it should be `nil`. As a workaround I check if user/password is `blank`, so it behaives correctly with empty strings.